package.json

```diff
@@ -55,7 +55,7 @@
     "es",
     "hydration",
     "devtools",
-    "persistQueryClient-experimental",
+    "persistQueryClient",
     "createWebStoragePersistor-experimental",
     "createAsyncStoragePersistor-experimental",
```

Comment:
those two should become stable as well please, as they go hand in hand with `persistQueryClient`.

and also renamed to `persister` please :)